api/controllers/v0.3/AddressValidationAPI.js

```diff
@@ -1,18 +1,16 @@
-/* eslint-disable */
 const ServiceObjectsClient = require("./ServiceObjectsClient");
 const { SONoLicenseKeyError } = require("../../helpers/errors");
 const { strToBool } = require("../../helpers/utils");
 
 /**
  * A class that uses Service Objects to validate addresses.
+ * @param {string} soLicenseKey License key for the Service Objects API.
  */
-const AddressValidationAPI = (args = {}) => {
-  const soLicenseKey = args["soLicenseKey"];
-
+const AddressValidationAPI = (soLicenseKey) => {
```

Comment:
For some classes, the object prop only had on attribute so it's just easier to pass that argument itself rather than in an object.